dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -280,24 +280,24 @@ BEGIN
     EDITTEXT 14019, 58, 163, 170, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
     LTEXT "", -1, 8, 179, 221, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
     LTEXT "Atribute:", 14020, 8, 189, 45, 10
-    AUTOCHECKBOX "Nem&odificabil", 14021, 58, 188, 67, 10
-    AUTOCHECKBOX "Asc&uns", 14022, 126, 188, 50, 10
-    AUTOCHECKBOX "Ar&hivabil", 14023, 181, 188, 49, 10
-    PUSHBUTTON "A&vansate…", 14028, 180, 185, 50, 15, WS_TABSTOP
+    AUTOCHECKBOX "&Doar în citire", 14021, 58, 188, 67, 10
+    AUTOCHECKBOX "&Ascuns", 14022, 126, 188, 50, 10
```

Comment:
23 minutes ago you changed this from
AUTOCHECKBOX "Asc&uns", 14022, 126, 188, 50, 10
to
AUTOCHECKBOX "&Ascuns", 14022, 126, 188, 50, 10
by
https://github.com/reactos/reactos/pull/6657/commits/04aff0705e31affbf17552fbc2229bfa2aed525b
Why?

In the IDD_FOLDER_PROPERTIES it is
AUTOCHECKBOX "Asc&uns", 14022, 126, 151, 55, 10
Does it really make any sense to do it different in both?

Reply:
Firstly, I thought I made a mistake, But now I checked again with the xp/2k3 file and discovered it was correct what I did before, so I reverted the change. I answered to you to a conversation you already marked as resolved.

Hence, I did a wrong job to put everywhere the accelerator in the same place.

Reply:
I was able to unresolve the conversation related to line 284. It seems it was correct before this is why I reverted the change.

Reply:
Ok, if MS does it differently also in both dialogs then that is fine. We are through now. I have no further complaints. I will approve now, and will wait 2 more days before I'll merge it. Please be aware that I will revoke my approval if anything more is added, that I don't like. So act wisely if you have further changes in mind. Also possible to just use the suggestion feature then, so things could be discussed before merging them into the PR. Thank you for your work.

Reply:
Thank you, also. Yes, this is the final form. So, no more changes will be made to this PR.